src/main/resources/anniv5Resources/localization/eng/dimensiongatepack/Cardstrings.json

```diff
@@ -19,14 +19,14 @@
   },
   "${ModID}:Knighthood": {
     "NAME": "Knighthood",
-    "DESCRIPTION": "Choose a card in hand to Upgrade permanently. NL Exhaust.",
+    "DESCRIPTION": "Ethereal. NL Choose a card in hand to Upgrade permanently. NL Exhaust.",
```

Comment:
With the new Ethereal -> not Ethereal upgrade, Knighthood needs an `UPGRADE_DESCRIPTION` entry added.